process_request/tests.py

```diff
@@ -267,6 +278,17 @@ def test_get_data(self):
         self.assertTrue(isinstance(get_as_data, list))
         self.assertEqual(len(get_as_data), 1)
 
+    @aspace_vcr.use_cassette("aspace_request.json")
+    @patch("asnake.client.web_client.ASnakeClient.get")
+    def test_invalid_get_data(self, mock_as_get):
+        error_message = "This is an error!"
```

Comment:
I know I did this but I think this should be a random string rather than a hardcoded one.